client/app/components/Input.jsx

```diff
@@ -18,12 +19,19 @@ export default class Input extends React.Component {
     this.props.collectData(this.state.trend);
   }
 
+  onEnter(event) {
+    console.log(event.key);
+    if( event.key === 'Enter' ){
```

Comment:
if you don't mind following the convention, the style here would be
`if (event.key === 'Enter') {`